Makefile

```diff
@@ -0,0 +1,19 @@
+SEVERITIES = HIGH,CRITICAL
+
+.PHONY: all
+all:
+	docker build --build-arg TAG=$(TAG) -t rancher/gke-operator:$(TAG) .
+
+.PHONY: image-push
+image-push:
+	docker push rancher/gke-operator:$(TAG) >> /dev/null
```

Comment:
What is the purpose of sending the output stream to `/dev/null`?

Reply:
to be honest, this was from the EKS, did not question what they did with that. happy not too as I can't think of a reason unless it was messing up something in the CI/CD (at least that is what I was assuming)

Reply:
Makes sense :) 
@rmweir do you know the origin of this?

Reply:
This was included when I generated the Makefile using: https://github.com/rancher/image-build-skel. @briandowns do you remember the reasoning for this? I imagine it would be CI/CD related or to just get rid clear up the logs.